interfaces/automated-testing/scd/design/strategic-coordination-interoperability/nominal-test.plantuml

```diff
@@ -0,0 +1,28 @@
+@startuml
+center header Strategic Coordination Interoperability
+title Nominal test
+
+participant "Test Driver" as TD
+participant "USS1 under test" as USS1
+participant "USS2 under test" as USS2
+participant "DSS"
+
+== TD injects via USS1 non-conflicting Operational Intent 1 ==
+
+TD --> USS1: PUT /inject/operational_intent
+TD <-- USS1: 200 OK - Operational Intent 1 created
+
+TD --> DSS: POST /dss/v1/operational_intent_references/query\nwith area_of_interest encompassing Operational Intent 1
+TD <-- DSS: 200 OK - returns Operational Intent Reference 1 with entityid1 managed by USS1
+
+TD --> USS1: GET /uss/v1/operational_intents/{entityid1}
+TD <-- USS1: 200 OK - returns Operational Intent 1
+
+...
+
+== TD injects via USS2 a conflicting Operational Intent (time, altitude, priority and area) ==
```

Comment:
We probably don't want to make a conflict on all 6 dimensions at the same time -- it would be better to propose many operational intents where each one conflicts on just one axis (to make sure the USS deconflicts on each axis).  In anticipation of that, I would drop the parenthetical here (just "TD injects via USS2 a conflicting Operational Intent"), or else use "or" instead of "and" when listing the possible conflict dimensions.